dl1_data_handler/utils.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import ctapipe.image.cleaning as cleaning
+from ctapipe.instrument import CameraGeometry
+
+
+#################
+# Event Filters #
+#################
+
+def event_energy_filter(reader, file, e_min=-np.inf, e_max=np.inf):
+    """
+    Filter events on energy
+    Parameters
+    ----------
+    reader
+    file
+    e_min
+    e_max
+
+    Returns
+    -------
+    the filtered indices
+    """
+    mask1 = (file.root.Events[:]['mc_energy'] > e_min) & (file.root.Events[:]['mc_energy'] < e_max)
+    return set(np.arange(len(file.root.Events))[mask1])
+
+
+def event_intensity_filter(reader, file, i_min=-np.inf, i_max=np.inf):
+    """
+    Filter events on intensity (in pe)
+    Parameters
+    ----------
+    reader (DL1DataReader) : the reader to filter
+    file
+    i_min
+    i_max
+
+    Returns
+    -------
+    the filtered indices
+    """
+    tel_types = [reader.tel_type] if reader.mode == 'mono' else reader.tel_type
```

Comment:
```suggestion
    tel_types = [reader.tel_type] if reader.mode == 'mono' else list(reader.selected_telescopes)
```

Reply:
Do you mean `reader.selected_telescope_types` ?

Reply:
No, this is a dictionary of tel_type: list of tel_ids (defined [here](https://github.com/cta-observatory/dl1-data-handler/blob/master/dl1_data_handler/reader.py#L159)).

Reply:
ok got it